trpcultivate_phenotypes/src/Service/TripalCultivatePhenotypesTermsService.php

```diff
@@ -3,26 +3,33 @@
 namespace Drupal\trpcultivate_phenotypes\Service;
 
 use Drupal\Core\Config\ConfigFactoryInterface;
-use Drupal\tripal_chado\Database\ChadoConnection;
 use Drupal\tripal\Services\TripalLogger;
+use Drupal\tripal_chado\ChadoBuddy\PluginManagers\ChadoBuddyPluginManager;
 
 /**
  * Phenotypes terms service.
  */
 class TripalCultivatePhenotypesTermsService {
   /**
-   * Configuration.
+   * The Chado Buddy cvterm.
    *
-   * @var config_entity
+   * @var object
    */
-  private $config;
+  protected object $cvterm_buddy;
```

Comment:
Always be specific about types ;-p

Also add a use statement for the buddies to ensure the second typehint works.

```suggestion
   * The Chado Buddy cvterm.
   *
   * @var Drupal\tripal_chado\Plugin\ChadoBuddy\ChadoCvtermBuddy
   */
  protected ChadoCvtermBuddy $cvterm_buddy;
```

Reply:
The code suggestion does not cause Problem and is committed in most recent commit.